src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -626,7 +626,7 @@ protected function extractDistinctValues($attribute)
      */
     public function validateEmail($attribute, $value)
     {
-        return (new EmailValidator)->isValid($value, new RFCValidation);
+        return is_string($value) && (new EmailValidator)->isValid($value, new RFCValidation);
```

Comment:
`RFCValidation::isValid()` also accepts an object implementing `__toString()`:

https://github.com/egulias/EmailValidator/blob/master/EmailValidator/Validation/RFCValidation.php#L30

Reply:
I change this. Now, the validation method will first check to have a string or an object implementing __toString, before validating it using RFC.